### PR TITLE
ogre3d: Add aarch64 support

### PR DESCRIPTION
**Pull request by hmartinez82**
Requires #12520 

image

### CODE REVIEW

**Comment by hmartinez82** [comment]
This can go ahead since it doesn't depend on the legacy directx files anymore.

File: mingw-w64-ogre3d/PKGBUILD [hunk 1]
```diff
@@ -99,6 +102,7 @@ build() {
     -DOGRE_BUILD_SAMPLES=ON \
     -DOGRE_INSTALL_SAMPLES=ON \
     -DOGRE_CONFIG_FILESYSTEM_UNICODE=ON \
+    $([[ ${MSYSTEM_CARCH} == aarch64 ]] && echo "-DOGRE_BUILD_RENDERSYSTEM_D3D9=OFF") \
```
Comment (Biswa96) [inline mingw-w64-ogre3d/PKGBUILD:105]: May I ask why this was disabled?

Comment (hmartinez82) [reply mingw-w64-ogre3d/PKGBUILD:105]: Windows on Arm doesn't support DirectX 9 natively. It has a redirection to dx 12 using a completely dll filename. Also mingw-w64 doesn't provide that import libraries for these different files,  the  d3dx*.dll ones.

Comment (hmartinez82) [reply mingw-w64-ogre3d/PKGBUILD:105]: The DirectX 11 renderer works perfectly :)

Comment (Biswa96) [reply mingw-w64-ogre3d/PKGBUILD:105]: If those **native arm64** version of d3dx*.dll files exist or provided by any MS package then those import libraries can be added in mingw-w64.

**Pull request merged**